.github/workflows/actualizar-eventos-meetup.yml

```diff
@@ -0,0 +1,64 @@
+name: "Actualizar eventos meetup"
+
+on:
+  schedule:
+    - cron: "0 12 * * 2" # Todos los martes a las 12:00
+  push:
+    paths:
+      - 'databags/meetup.json'
+
+jobs:
+  build:
+    name: Actualizar eventos meetup
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout@v3
+        uses: actions/checkout@v3
+        with:
+          ref: master
+
+      - name: setup-python@v4
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+
+      - name: Instalar requirements
+        run: |
+          python -m pip install --upgrade pip setuptools
+          pip install -r requirements.txt
+
+      - name: Ejecutar events.py
+        run: python scripts/events.py
+
+      - name: Rama actual
+        run: git branch
+
+      - name: Git status
+        run: git status
+
+      - name: Verificar cambios en meetup.json
+        id: check_changes
+        # https://git-scm.com/docs/git-diff#Documentation/git-diff.txt---exit-code
+        run: |
+          if git diff --quiet --exit-code -- databags/meetup.json; then
+        # https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter
+            echo "HAS_CHANGES=false" >> "$GITHUB_OUTPUT" 
+            echo "No hay cambios en meetup.json"
+          else
+            echo "HAS_CHANGES=true" >> "$GITHUB_OUTPUT"
+          fi
+
+      - name: Git crear rama y commit
+        if: steps.check_changes.outputs.HAS_CHANGES == 'true'
+        run: |
+          git config --global user.name "PyBaq Bot"
+          git config --global user.email "contacto@pybaq.co"
+          git checkout -b actualizar-eventos-meetup
```

Comment:
Seria muy genial tener un parametro que nos permita escoger el nombre de la rama

Reply:
Puede ser un nombre de rama con la fecha y hora

Reply:
¿Ese input como string para el nombre de la rama en qué momento pide esa información?

Reply:
es un parametro de entrada cuando uses un disparador llamado workflow dispatch, y se le puede colocar un valor por defecto 
https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/

Reply:
```yaml
          git checkout -b {{ github.event.inputs.branch }}
```

Reply:
Hola, no entiendo que quisiste decir con `git checkout -b {{ github.event.inputs.branch }}`